README.md

```diff
@@ -150,6 +150,7 @@ The ilib-assemble tool takes the following options:
     the default (`./src/ilib-all-inc.js`)file will be used.
 * --outjsFileName or n. Specify the resulting assembled output file name.
     The default is `ilib-all.js`
+* -- customLocalePath or p. Specify the location where the custom locale data path. If you want to override the value of existing locale data.
```

Comment:
->

"--customLocalePath or p. Specify the path to customized locale data that overrides existing open-source locale data."

Reply:
I update it